docs/platform/concepts/corporate-billing.rst

```diff
@@ -0,0 +1,49 @@
+Corporate Billing
+=================
+
+Everyone has their preferred way of paying for things. Here at Aiven we give you the flexibility to either recieve invoices or paying by credit card for your Aiven services.
+
+The **default billing** method for all new Aiven customers is a credit card charge at the end of the month for all charges accrued over that month. Each Aiven project is charged separately.
+
+The billing unit for Aiven services is 1 hour. We automatically calculate the amount based on the services that have been running in your Aiven project. 
+
+For customers who prefer to pay by invoice or are unable to pay with a credit card, we also offer corporate billing options.  The standard corporate billing option is monthly invoices generates at the end of the month based on actual usage, similar to the credit card charge method used by default.
+
+These invoices are automatically generated by our backend and sent as PDF over email to the billing email addresses configured in the Aiven dashboard.  
+
+Prices for Aiven services are always defined in US dollars, but invoices can be sent in other currencies using the invoice date's currency exchange rates.  Currently, the following currencies are accepted:
+
+-  US dollars (default)
+
+-  Euros
+
+-  Pounds sterling 
+
+-  Swiss francs
+
+-  Danish kroner
+
+-  Norwegian kroner
+
+-  Swedish kronor
+
+-  Australian dollars 
+
+-  Canadian dollars 
+
+A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction. Invoice must be paid within **10 days** of receipt by a wire transfer to Aiven bank account at Citibank Europe Plc:
```

Comment:
This line is not very clear to me. Maybe rewrite it?

"A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction."

Reply:
I'm deleteting this line "A payment due upon receipt is a payment that customers must make immediately upon receiving the invoice for a transaction."